helper-cli/src/main/kotlin/commands/packagecuration/SplitCommand.kt

```diff
@@ -68,27 +70,32 @@ internal class SplitCommand : CliktCommand(
 
         val mapper = createBlockYamlMapper()
         groupedCurations.forEach { (outputFile, curations) ->
+            val curationsToPersist = if (outputFile.isFile) {
+                outputFile.readValue<List<PackageCuration>>()
+            } else {
+                emptyList()
+            }.toMutableList()
+
             curations.forEach { originalCuration ->
                 val comment = originalCuration.data.comment?.wrapAt(COMMENT_WRAP_COLUMN)
 
-                val curation = if (comment != null) {
+                curationsToPersist += if (comment != null) {
                     // Ensure at least a single "\n" is contained in the comment to force the YAML mapper to use block
                     // quotes.
                     originalCuration.copy(data = originalCuration.data.copy(comment = "$comment\n"))
                 } else {
                     originalCuration
                 }
+            }
 
-                val text = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(listOf(curation))
-
-                outputFile.parentFile.safeMkdirs()
+            val text = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(curationsToPersist.sortedBy { it.id })
```

Comment:
Should sorting work on each component of the idea separately?

Reply:
I'm sorry, I don't understand your question. Can you please rephrase?

Reply:
I edited above comment, there was a typo in.

Reply:
Thanks for the clarification.
You're right it is not required to use the default comparator from `Identifier`.
I changed it to just sort by `it.id.version` to make this clear (even if the functionality is not changed).